Firestore/Example/Tests/SpecTests/FSTMockDatastore.mm

```diff
@@ -248,6 +239,13 @@ int SentMutationsCount() const {
 }  // namespace firestore
 }  // namespace firebase
 
+using firebase::firestore::remote::ConnectivityMonitor;
```

Comment:
(using decl's in the middle of the file)

Reply:
Moved all of them to the top, except for `MockWatchStream`/`MockWriteStream`.